README.md

```diff
@@ -1,13 +1,34 @@
 # graph-cosmos-ts
-Graph cosmos protobuf typescript binding
-
-| Directory              | Source                                                                           |
-|------------------------|----------------------------------------------------------------------------------|
-| `confio`               | https://github.com/confio/ics23/blob/v0.6.6/proofs.proto                         |
-| `cosmos`               | https://github.com/cosmos/cosmos-sdk/tree/v0.45.1/proto/cosmos                   |
-| `cosmos_proto`         | https://github.com/cosmos/cosmos-proto/tree/v1.0.0-alpha7/proto/cosmos_proto     |
-| `gogoproto`            | https://github.com/regen-network/protobuf/tree/v1.3.3-alpha.regen.1/gogoproto    |
-| `google/api`           | https://github.com/googleapis/googleapis/tree/master/google/api                  |
-| `ibc`                  | https://github.com/cosmos/cosmos-sdk/tree/v0.42.10/proto/ibc                     |
-| `protoc-gen-openapiv2` | https://github.com/grpc-ecosystem/grpc-gateway/tree/v2.10.0/protoc-gen-openapiv2 |
-| `tendermint`           | https://github.com/tendermint/tendermint/tree/v0.34.14                           |
+
+This modules contains the typescript/assemblyscript binding for cosmos protobuf
+
+## Development
+
+Install protoc: `https://github.com/protocolbuffers/protobuf/releases`
+
+### Get proto-file
+
+```bash
+./submodule-init.sh
+git submodule update --force
```

Comment:
`git submodule update --init --recursive`?

Reply:
`git submodule update --force` is only what is needed when using `./submodule-init.sh`